### PR TITLE
🔨 📝 Maintenance: code-ownership, citations and conventions [skip CI]

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- 📝 Propose naming conventions for  factory fixtures (See `CC3`). Please comment!
- 📝 updates citation list for new contributors
- 🔨updates CODEOWNERSHIP of front-end (after @odeimaiz left for :palm_tree: ). **REMEMBER**: @ignapas and @jsaq007 own now (i.e. always **required** to review) all changes in the front-end

## Related issue/s

None

## How to test

None

## DevOps 
NOne

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -35,7 +35,7 @@ Makefile                                  @pcrespov @sanderegg
 /services/payments/                       @pcrespov
 /services/resource-usage-tracker/         @matusdrobuliak66
 /services/static-webserver/               @GitHK
-/services/static-webserver/client/        @odeimaiz
+/services/static-webserver/client/        @jsaq007 @ignapas
```
Comment (GitHK) [inline .github/CODEOWNERS:38]: 👍

**Review by YuryHrytsuk — APPROVED** [review]
Thank you!

File: docs/coding-conventions.md [hunk 2]
```diff
@@ -37,7 +37,30 @@ import os
 # x = "ok"
 ```
 
-### CC3 ...
+### CC3: Naming fixtures that return `Callable`s
+
+- Use verb as prefix
+- If the callable creates an instance, you can use `factory_` as suffix instead (e.g. the [`tmp_path_factory` fixture](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture))
```
Comment (GitHK) [inline docs/coding-conventions.md:43]:
```suggestion
- If the callable creates an instance, you can use `_factory` as suffix instead (e.g. the [`tmp_path_factory` fixture](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture))

```

File: docs/coding-conventions.md [hunk 3]
```diff
@@ -37,7 +37,30 @@ import os
...
+
+- Use verb as prefix
+- If the callable creates an instance, you can use `factory_` as suffix instead (e.g. the [`tmp_path_factory` fixture](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture))
+
+```python
+
+@pytest.fixture
+def something() -> Something:
+  return Something("nice")
+
```
Comment (matusdrobuliak66) [inline docs/coding-conventions.md:50]: :+1:

**Review by matusdrobuliak66 — APPROVED** [review]

File: docs/coding-conventions.md [hunk 4]
```diff
@@ -37,7 +37,30 @@ import os
 # x = "ok"
 ```
 
-### CC3 ...
+### CC3: Naming fixtures that return `Callable`s
+
+- Use verb as prefix
```
Comment (GitHK) [inline docs/coding-conventions.md:42]: I would put this line below the next one. It's a bit confusing when reading

**Review by GitHK — APPROVED** [review]

File: docs/coding-conventions.md (hunk 4 above)
Comment (pcrespov) [reply docs/coding-conventions.md:42]: @GitHK I am not sure what you mean. Can you please provide an edit as before? that would be easier

**Review by mrnicegyu11 — APPROVED** [review]

File: docs/coding-conventions.md [hunk 5]
```diff
@@ -37,7 +37,31 @@ import os
 # x = "ok"
 ```
 
-### CC3 ...
+### CC3: Naming fixtures that return `Callable`s
+
+- Use verb as prefix
+- If the callable creates an instance, you can use `_factory` as suffix instead (e.g. the [`tmp_path_factory` fixture](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture))
```
Comment (GitHK) [inline docs/coding-conventions.md:43]: this is what I mean

```suggestion
- If the callable creates an instance, you can use `_factory` as suffix instead (e.g. the [`tmp_path_factory` fixture](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture))
- Use verb as prefix, as seen in the example below
```

**Review by GitHK — APPROVED** [review]

**Review by bisgaard-itis — APPROVED** [review]

File: docs/coding-conventions.md [hunk 6]
```diff
@@ -37,7 +37,31 @@ import os
...
+
+
+```python
+
+@pytest.fixture
+def something() -> Something:
+  return Something("nice")
+
+@pytest.fixture
+def create_something() -> Callable:
```
Comment (sanderegg) [inline docs/coding-conventions.md:53]:
```suggestion
def create_something() -> Callable[..., Something]:

```

File: docs/coding-conventions.md [hunk 7]
```diff
@@ -37,7 +37,31 @@ import os
...
+  def _create(*args, **kwargs):
```
Comment (sanderegg) [inline docs/coding-conventions.md:54]:
```suggestion
  def _create(*args, **kwargs) -> Something:

```

File: docs/coding-conventions.md [hunk 8]
```diff
@@ -37,7 +37,31 @@ import os
...
+    # ...
+  return _create
+
+def test_it(something: Something, create_something: Callable):
```
Comment (sanderegg) [inline docs/coding-conventions.md:58]:
```suggestion
def test_it(something: Something, create_something: Callable[..., Something]):

```

**Review by sanderegg — APPROVED** [review]
I agree with you, but please add the typing. This makes everything pylance-related and so on much more easy to use.

**Pull request merged**